labonneboite/web/admin/views/office_admin_update.py

```diff
@@ -10,6 +10,14 @@
 from labonneboite.web.admin.utils import datetime_format, AdminModelViewMixin
 from labonneboite.conf import settings
 
+def is_siret(siret):
+    # A valid SIRET is composed by 14 digits
+    try:
+        int(siret)
+    except ValueError:
+        return False
+
+    return len(siret) == 14
```

Comment:
pour info, c'est tout comme : `return len(siret) == 14 and siret.isdigit()`

Reply:
:+1:

Reply:
`labonneboite/web/admin/forms.py:11:    if not field.data.isdigit() or len(field.data) != 14:`